src/binding-signaler.js

```diff
@@ -1,16 +1,15 @@
-import {sourceContext} from 'aurelia-binding';
+import {sourceContext, signalBindings} from 'aurelia-binding';
+import * as LogManager from 'aurelia-logging';
 
 export class BindingSignaler {
-  signals = {};
 
+  constructor() {
+    this.logger = LogManager.getLogger('binding-signaler');
+  }
+  
   signal(name: string): void {
-    let bindings = this.signals[name];
-    if (!bindings) {
-      return;
-    }
-    let i = bindings.length;
-    while (i--) {
-      bindings[i].call(sourceContext);
-    }
+    this.logger
+      .warn(`Binding signaler signal has been deprecated. Please use "import {signalBindings} from 'aurelia-binding' instead.`);
```

Comment:
I was thinking this would move to the constructor as well. This way the message appears only once, the first time the application instantiates the BindingSignaler.  Sorry I wasn't clear the first time.